continuous_integration/recipes/distributed/meta.yaml

```diff
@@ -2,19 +2,23 @@
 {% set new_patch = major_minor_patch[2] | int + 1 %}
 {% set version = (major_minor_patch[:2] + [new_patch]) | join('.') + environ.get('VERSION_SUFFIX', '') %}
 {% set dask_version = environ.get('DASK_CORE_VERSION', '0.0.0.dev') %}
-{% set build_ext = "cython" %}  # [cython_enabled]
-{% set build_ext = "python" %}  # [not cython_enabled]
 
 
 package:
-  name: distributed-split
+  name: distributed
   version: {{ version }}
 
 source:
   git_url: ../../..
 
 build:
   number: {{ GIT_DESCRIBE_NUMBER }}
+  string: py{{ python | replace(".", "") }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - dask-scheduler = distributed.cli.dask_scheduler:go
+    - dask-ssh = distributed.cli.dask_ssh:go
+    - dask-worker = distributed.cli.dask_worker:go
 
 outputs:
   - name: distributed-impl
```

Comment:
We no longer need `distributed-impl` now that we aren't doing cython builds correct?

If so, we should probably replace everything below `source` with the contents of the `distributed` output

Reply:
I fixed the other points, above, but I didn't understand what to do here.

Reply:
Yeah I realize I probably didn't explain this the best 😅 opened a PR against your branch doing what I described:

- https://github.com/martindurant/distributed/pull/1